src/Module/Contact.php

```diff
@@ -293,7 +293,7 @@ public static function content(array $parameters = [], $update = 0)
 			$contact = DBA::selectFirst('contact', [], ['id' => $contact_id, 'uid' => [0, local_user()], 'deleted' => false]);
 
 			// Don't display contacts that are about to be deleted
-			if ($contact['network'] == Protocol::PHANTOM) {
+			if (DBA::isResult($contact) && !empty($contact['network']) && $contact['network'] == Protocol::PHANTOM) {
```

Comment:
Please check instead if the array `$data` is empty and only then fetch the `$contact` array.

Reply:
I added both  `if`s for safety reason ;-)